tools/auto_placement/README.md

```diff
@@ -0,0 +1,65 @@
+# Auto Placement Tool
+
+This tool automatically generates optimal component placement configurations for RL training workflows. It analyzes the computational costs of different components (rollout, inference, training, etc.) and determines the best placement strategy to minimize overall training time.
+
+## Overview
+
+The auto placement tool consists of three main components:
+
+- **`scheduler_task.py`**: Main scheduler that performs time and space division multiplexing to find optimal placements
+- **`resource_allocator.py`**: Handles resource allocation for different components
+- **`workflow.py`**: Manages workflow graphs and cost calculations
+
+## Usage
+
+### Step 1: Collect Profile Data
+
+Before running the auto placement tool, you need to collect profile data for your components. This includes measuring the computation time for each component (rollout, inference, training, etc.) in colocate mode for one iteration.
+
+Add the profile data to your YAML configuration file under the `profile_data` section:
+
+```yaml
+profile_data:
+  trainer_cost: 95.7    # Training component cost (seconds per iteration)
+  inference_cost: 30.8  # Inference component cost (seconds per iteration)
+  rollout_cost: 59.9    # Rollout component cost (seconds per iteration)
+```
+
+**How to collect profile data:**
+1. Run your training with origin cluster in colocate mode for several iterations
+2. Use profiling tools to measure the time each component takes per iteration
+3. Record the average time per iteration for each component
+
+### Step 2: Run Auto Placement
+
+Use the provided shell script to run the auto placement tool:
+
+```bash
+cd examples/math
+./run_placement_autotune.sh [config_name]
+```
+
+Where `config_name` is the name of your configuration file (default: `qwen2.5-1.5b-grpo-megatron`).
```

Comment:
better not to add default file name, users should explicitly specify the config file name.